avocado/utils/linux_utils.py

```diff
@@ -0,0 +1,51 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# This code was inspired in the autotest project,
+#
+# client/base_utils.py
+#
+# Copyright: 2019 IBM
+# Authors : Praveen K Pandey <praveen@linux.vnet.ibm.com>
+
+
+"""
+Linux OS  utilities
+"""
+
+import re
+
+from . import genio
+
+
+def sysctl(key, value=None):
```

Comment:
I find it highly uncommon to have a single function for both setting and getting a value... I honestly think I can see things from your perspective, in the sense that this is such a simple function that it's not worth having two.

But, one of the Python motto (in the Zen of Python) is that explicit is better than implicit.  So, I'd suggest you to define both a `get_proc_sys()` and `set_proc_sys()` types of function.

Reply:
@clebergnu yes make sense will adopt this